wp-content/plugins/core/src/Blocks/Types/Post_List/Post_List.php

```diff
@@ -10,35 +10,15 @@
 use Tribe\Libs\ACF\Repeater;
```

Comment:
I would really like to propose that we delete this Block entirely. It's not a "real" block as defined by the designs. It was build as a demo for the Post List field (Query) functionality. The purpose of this block is generally confusing for developers on projects.

Reply:
Good call, agreed